spec/mailers/provider_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe ProviderMailer, type: :mailer do
+  describe "waiting_administrator_confirmation" do
```

Comment:
don't forget to add `.` if it is a class method, or `#` if it is an instance method.